shpc/main/loader.py

```diff
@@ -0,0 +1,67 @@
+__author__ = "Vanessa Sochat"
+__copyright__ = "Copyright 2021-2022, Vanessa Sochat"
+__license__ = "MPL 2.0"
+
+from jinja2.exceptions import TemplateNotFound
```

Comment:
@muffato sorry haven't looked at this in a bit - didn't we replace this with a simpler solution?

Reply:
oops my bad, wasn't merged yet - here it is: https://github.com/muffato/singularity-hpc/pull/3